composition/stylesheet_deduplication.go

```diff
@@ -0,0 +1,53 @@
+package composition
+
+import (
+	"strings"
+
+	"golang.org/x/net/html"
+)
+
+// Initialization: Sets the default deduplication strategy to be used.
+func init() {
+	SetStrategy(new(IdentityDeduplicationStrategy))
+}
+
+// Set another deduplication strategy.
+func SetStrategy(strategy StylesheetDeduplicationStrategy) {
+	stylesheetDeduplicationStrategy = strategy
+}
+
+// NOOP strategy.
+// This stragegy will insert all found stylesheets w/o any filtering.
+type IdentityDeduplicationStrategy struct {
```

Comment:
Confusing name... Maybe something more expressive in regards of it being a noop.